cli/tests/activate.bats

```diff
@@ -924,3 +924,23 @@ EOF
 }
 
 # ---------------------------------------------------------------------------- #
+
+# bats test_tags=activate,activate:zdotdir,activate:zdotdir:bash
+@test "bash: preserve custom ZDOTDIR" {
```

Comment:
question nonblocking: 🤔 do we touch ZDOTDIR for a bash activation?

Reply:
We don't, but thought it seemed a reasonable test to ensure we don't inadvertently set the variable - you never know when someone will launch zsh as a child process of a bash invocation.